_posts/2017-04-04-things-the-rust-community-team-does.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Things the Rust Community Team does"
+author: Florian Gilcher
+description: "Describing the Rust Community Team work"
+---
+
+A common question that we get is "what precisely does the Rust community team do?" and probably the member you are currently speaking to has no straight answer to you. That has a reason: the team does a lot of things and any individual often just does a couple of them. Many of the projects that come out of this are then taken on by individual teams and are not necessarily visible as community team action. An important part is to discuss our and others ideas and support each other in our endeavours.
+
+Still, where there's no information, misconceptions arise, in all directions: either that the team doesn't do anything at all or that the teams size and work is way exaggerated.
+
+So, I recently accidentally wrote a pretty exhausive list, which I'd like to extend and republish here. A lot of what we do is all over the place and in a DIY fashion, so we learn a lot of skills on the go. If you have experience with any of these - please get involved!
```

Comment:
Could we have `get involved` be an actionable link to the community issues or the discourse category? or probably a `mailto:`?